typespec-extension/src/code-model-builder.ts

```diff
@@ -345,14 +345,18 @@ export class CodeModelBuilder {
             {
               const schemeOrApiKeyPrefix = scheme.scheme;
               if (schemeOrApiKeyPrefix === "basic" || schemeOrApiKeyPrefix === "bearer") {
-                this.logWarning(`{scheme.scheme} auth method is currently not supported.`);
-              } else {
-                const keyScheme = new KeySecurityScheme({
-                  name: "authorization",
-                });
-                (keyScheme as any).prefix = schemeOrApiKeyPrefix; // TODO (weidxu): modify KeySecurityScheme, after design stable
-                securitySchemes.push(keyScheme);
+                if (!(this.options.branded === false)) {
```

Comment:
May consider remove the branch.

We'd expect lint would catch this in Azure service.